VideoLocker/src/main/java/org/edx/mobile/module/registration/view/RegistrationAgreementView.java

```diff
@@ -31,13 +32,13 @@ public RegistrationAgreementView(RegistrationFormField field, View view) {
 
         // display label as HTML and text to be centered horizontally
         mInputView.setGravity(Gravity.CENTER_HORIZONTAL);
-        mInputView.setText(mField.getAgreement().getText());
-        mInputView.setOnClickListener(new View.OnClickListener() {
 
+        mInputView.setText(R.string.tos_and_honor);
```

Comment:
The label from the JSON file does not match what we want. I floated the
idea of modifying what was returned from the server but the team didn’t
like that (that API is used by the web too).

On Tue, Sep 13, 2016 at 4:11 PM Brian @ edX <notifications@github.com>
wrote:

> In
> VideoLocker/src/main/java/org/edx/mobile/module/registration/view/RegistrationAgreementView.java
> <https://github.com/edx/edx-app-android/pull/828#discussion_r78634738>:
>
> >
> > +        mInputView.setText(R.string.tos_and_honor);
>
> Why do we need to hard code this instead of using the label in the JSON?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/edx/edx-app-android/pull/828/files/6672e9ed98a716a94f8211853f7eb2d4fcbc8e03#r78634738>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AR10Tyd9QcYcaDqeHf6SuUwqbtQJhvZ6ks5qpwN2gaJpZM4J7vAI>
> .
>